### ISSUE TITLE
Inclusion in F-Droid

### ISSUE THREAD

**Issue by robinpaulson** [qpython-android/qpython#20]
Now that all the repositories have licences, we can request it be included in F-Droid.

I did so a few minutes ago, here is the link to the ticket:
https://gitlab.com/fdroid/rfp/issues/239

There will likely be questions about the inclusion, around things including pre-built libraries (if any), build scripts, update methods and so on. Your assistance answering questions would be appreciated.

Cheers

**Comment by ale5000-git** [comment]
+1000

I really like this project but I think that it won't happen since F-Droid require all binaries to be compiled from sources and this repository doesn't contain sources of everything and I think (I really hope to be wrong) that developers are un-willingly to allow others to compile the project completely.

Ref: https://github.com/qpython-android/qpython3/issues/2

**Comment by robinpaulson** [comment]
@ale5000-git Yes, I think this too. However, I've put some effort into finding the original sources of the binaries, so we can get around qpython devs dragging their heels. I won't let that hold it up, the biggest holdup at the moment appears to be workload/backlog of f-droid devs to include new apps.

**Comment by riverfor** [comment]
In fact , the QPython community brand's sourcecode are here, but they don't be integrated well (for the work is a bit complex ),

1 Android sourcecode
https://github.com/qpython-android/qpython

2 QPython's core code are here 
https://github.com/qpython-android/qpython-core

Now, we build the QPython binary core from 
https://github.com/qpython-android/qpython-core

then build the android APK with
https://github.com/qpython-android/qpython

The two steps are done manually,

What can I help ?

**Comment by robinpaulson** [comment]
@riverfor At the moment, any questions/problems are listed on the issue for the inclusion:
https://gitlab.com/fdroid/rfp/issues/239
If you could keep an eye on that, that would be great. I will also post messages and questions here as they come up.

**Comment by robinpaulson** [comment]
First question: is there a way to build the app without Umeng?

**Comment by eghil88** [comment]
Ok thanks

**Comment by riverfor** [comment]
Can you send me( river@qpython.org) your contact's telegram to me, may be we could chat with telegram to solve this more efficiently ?
